draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -89,6 +89,13 @@ normative:
       ins: D. Denicola
       name: Domenic Denicola
       organization: Google, Inc.
+  SAMESITE:
+    target: https://html.spec.whatwg.org/#same-site
+    title: HTML - Living Standard
+    date: 2021-01-26
```

Comment:
```suggestion
```

Reply:
@reschke Our hope here is to capture the context of the time when the HTML spec was referenced, akin to how one would use `Accessed: <date>` when citing a web reference in other contexts. There is some precedence for this in other recent IETF documents: https://tools.ietf.org/html/rfc8833#ref-HTML5, but I also see documents that leave the date off (e.g., https://raw.githubusercontent.com/httpwg/http-extensions/master/draft-ietf-httpbis-bcp56bis.md).

Is the prevailing norm that no dates are included in references to living standards? If so, I won't object to dropping it.

Reply:
We reached out to folks from the RFC Production Center (RPC) for guidance on how to reference HTML. Their guidance is: `We recommend including a date if the intent is to refer to a specific version in time or if the text refers to a specific section within a document.  If the goal is for readers to refer the most current version of the document at any given time, we still suggest including the date but it is not required.`. This reference clearly meets the first use case, and as such we should keep a date here.

Reply:
@reschke thoughts on @englehardt's last comment? If you're happy with it, can you update your review (to not request changes)? Thanks.

Reply:
I'm going to merge this now since we're blocking a PR for #1337. Feel free to file a follow-up issue if there's more to discuss!